redfish-core/lib/account_service.hpp

```diff
@@ -1288,177 +1288,187 @@ inline void handleLDAPPatch(nlohmann::json& input,
-                persistent_data::SessionStore::getInstance()
-                    .removeSessionsByUsernameExceptSession(username, session);
-                messages::success(asyncResp->res);
-            }
+            // Remove existing sessions of the user when password changed
+            persistent_data::SessionStore::getInstance()
+                .removeSessionsByUsernameExceptSession(username, session);
+            messages::success(asyncResp->res);
         }
+    }
 
-        if (mfaBypass)
+    if (mfaBypass)
+    {
+        std::string mfaBypassDbusVal;
+        // Check if the input vector contains just one bypass type:
+        // as there are just two defined in the backend:
+        // GoogleAuthenticator  and None
+        if (username == "service")
+        {
+            messages::operationNotAllowed(asyncResp->res);
+            return;
+        }
+        if (mfaBypass->size() > 1)
+        {
+            std::string values = std::accumulate(
+                std::next(mfaBypass->begin()), mfaBypass->end(),
+                mfaBypass->front(),
+                [](const std::string& str1, const std::string& str2) {
+                return str1 + ", " + str2;
+            });
+            messages::propertyValueIncorrect(asyncResp->res,
+                                             "MFABypass/BypassTypes", values);
+            return;
+        }
+        std::string mfaBypassDbusPrefix =
+            "xyz.openbmc_project.User.MultiFactorAuthConfiguration.Type.";
+        if (mfaBypass->empty())
         {
-            std::string mfaBypassDbusVal;
-            // Check if the input vector contains just one bypass type:
-            // as there are just two defined in the backend:
-            // GoogleAuthenticator  and None
-            if (username == "service")
+            mfaBypassDbusVal = mfaBypassDbusPrefix + "None";
+        }
+        else
+        {
+            std::string mfaBypassVal = mfaBypass->front();
+            if (mfaBypassVal == "GoogleAuthenticator" || mfaBypassVal == "None")
             {
-                messages::operationNotAllowed(asyncResp->res);
-                return;
+                mfaBypassDbusVal = mfaBypassDbusPrefix + mfaBypassVal;
             }
-            if (mfaBypass->size() > 1)
+            else
             {
-                messages::propertyNotWritable(asyncResp->res,
-                                              "MFABypass/BypassTypes");
+                messages::propertyValueNotInList(asyncResp->res, mfaBypassVal,
+                                                 "MFABypass/BypassTypes");
                 return;
             }
-            std::string mfaBypassDbusPrefix =
-                "xyz.openbmc_project.User.MultiFactorAuthConfiguration.Type.";
-            if (mfaBypass->empty())
-            {
-                mfaBypassDbusVal = mfaBypassDbusPrefix + "None";
-            }
-            else
+        }
+        sdbusplus::asio::setProperty(
```

Comment:
This can be done by `setDbusProperty()` like what's done in upstream, as 1110 also has the function.

Reply:
Done.